slack_sdk/oauth/installation_store/models/installation.py

```diff
@@ -190,6 +190,11 @@ def to_dict(self) -> Dict[str, Any]:
             "token_type": self.token_type,
             "installed_at": datetime.utcfromtimestamp(self.installed_at),
         }
+
+    def to_dict_for_copying(self) -> Dict[str, Any]:
```

Comment:
In the next major version of this project should we remove these changes and make `to_dict` behave like `to_dict_for_copying`?
If so should we add a `TODO` comment here that informs maintainer to do this in the next major?

Reply:
Thanks for the suggestion! Changing the behavior may break existing apps, so I personally don't think we should modify to_dict method. For instance, your custom InstallationStore may generate a dict object using the method and save it in simple data storage like Amazon S3. That said, if our future maintainers decide to change for a certain reason, I don't oppose to it at all.